json-spec/extensions/stac-eo-spec.md

```diff
@@ -0,0 +1,60 @@
+| eo:collection             | string                      | Collection Name       | The name of the STAC `Collection` this `Item` belongs to |
+| eo:crs     | reference system    | ref system             | CRS of the datasource in full WKT format. null if no crs
+| eo:cloud_cover     | number (optional)   | Cloud Cover Pct    | Percent of cloud cover | 
+| eo:off_nadir      | number (optional)   | Off nadir    | The viewing angle measured from nadir, in degrees
+| eo:azimuth      | number (optional)   | Azimuth    | The viewing azimuth angle, in degrees
+| eo:sun_azimuth    | number (optional)   | Sun Azimuth | The sun azimuth angle, in degrees
+| eo:sun_elevation  | number (optional)   | Sun Elevation | The sun elevation angle, in degrees
+| eo:supported_assets | array of asset names | Supported Assets | A list of the key names for the asset types that are available now.
+
+### `Item:assets` additions
+
+| element             | type info                 | name                    | description                                                                                 | 
+|----------------------|---------------------------|-------------------------|---------------------------------------------------------------------------------------------| 
+| bands | string array | Bands | An array of strings that index the `eo:bands` field in a `Collection`.
+
+### `Collection` additions
+In addition, the EO Extension defines additional fields under a STAC `Collection`.
+| element             | type info                 | name                    | description                                                                                 | 
+|----------------------|---------------------------|-------------------------|---------------------------------------------------------------------------------------------| 
+| eo:platform            | string                      | Unique name of platform | Specific name of the platform (e.g., landsat-8, sentinel-2A, larrysdrone) | 
+| eo:instrument        | string                      | Instrument used     | Name of instrument or sensor (e.g., MODIS, ASTER, OLI, Canon F-1) |
+| eo:bands               |   dict                       | Bands                    | A dictionary of band properties |
+
+#### `Collection:bands`
+The bands field of a `Collection` is a dictionary where the index identifies a specific band. This is often a band number (e.g., 1, B1, B01), but could be any unique identifier.
+| element             | type info                 | name                    | description                                                                                 | 
+|----------------------|---------------------------|-------------------------|---------------------------------------------------------------------------------------------| 
+| common_name | string (optional) | Common name | The name commonly used to refer to this specific band (e.g., red, blue, nir)
```

Comment:
Should all these fields be optional? Seems like there should be at least one or two required elements to be useful.

Reply:
common_name really should be an enumeration of accepted values that describe the band in terms of a standard spectral band within the EO spectrum. I have a start with the 'Common Band Names' table in the stac-eo-spec.md file here which I think includes most of the common_names that we would want. The idea here is to be able to know which bands across sensors are equivalent.
This is most important with RGB and NIR bands, slightly less so with the two major SWIR windows at 1.6um and 2.2um, but still important to know, e.g. Landsat-8 SWIR bands vs Sentinel-2 SWIR bands.
Long wave IR (LWIR) is even less common. Most LWIR bands will be anywhere between 1/2 - 1 micron in width somewhere between the atmospheric window of 8 and 12 micron.

If common_name ends up becoming a string it really defeats the purpose of having a common_name, it really should belong to a set of values which should at least include RGB+NIR+SWIR16+SWIR22 (SWIR bands in the 1.6 and 2.2 micron windows).